api/v1/runtimecomponent_types.go

```diff
@@ -405,6 +405,9 @@ type RuntimeComponentStatus struct {
 	Binding *corev1.LocalObjectReference `json:"binding,omitempty"`
 
 	References common.StatusReferences `json:"references,omitempty"`
+
+	// The last generation of this RuntimeComponent instance observed by the operator.
```

Comment:
Suggestion: The generation identifier of this RuntimeComponent instance completely reconciled by the Operator.

Similar comment for the other instances

Reply:
Updated the descriptions